src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Owin/OwinStartupMiddleware.cs

```diff
@@ -56,7 +56,7 @@ public override async Task Invoke(IOwinContext context)
 
             try
             {
-                await _next.Invoke(context);
+                await _next.Invoke(context).ConfigureAwait(false);
```

Comment:
I don't know if this would cause problems with other middelwares if a synchronization is present.